### ISSUE TITLE
SwitchButtonView text should not change color when it is on

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#12519]
## 📝 Provide detailed reproduction steps (if any)

In https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/examples/builds-custom/full-featured-editor.html.

1. Create a numbered list.
1. Open the numbered list properties drop-down.
1. Toggle the reversed order switch

### ✔️ Expected result

The text stays black. The moving colorful switch is enough to carry the the state change.

### ❌ Actual result

![2022-09-27 16 38 33](https://user-images.githubusercontent.com/1099479/192557241-efb8f495-dd43-4054-943b-825cea5dda8d.gif)

### ❓ Possible solution

I suppose we broke it while working on the a11y of the interface (icons getting a blueish color when the button is on). It's funny though, this bug is not visible in link decorator feature and find and replace settings where `SwitchButtonView` is also used. This is probably due to the fact that these features bring their own styles that (for whatever the reason) override the text color of switch buttons.

Long story short:
1. This needs to be fixed at `switchbutton.css` level in theme-lark.
2. It would be great if we knew why link decorators and find and replace are not affected. Maybe there's something else going on we don't understand?

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
